pages.de/common/nix-shell.md

```diff
@@ -0,0 +1,28 @@
+# nix-shell
+
+> Startet eine interaktive Shell, die auf einem Nix-Ausdruck basiert.
+> Weitere Informationen: <https://nixos.org/manual/nix/stable/#sec-nix-shell>.
+
+- Start mit Nix-Ausdruck in `shell.nix` oder `default.nix` im aktuellen Verzeichnis:
+
+`nix-shell`
+
+- Shell-Befehl in nicht-interaktiver Shell ausführen und beenden:
+
+`nix-shell --run "{{Befehl}} {{Befehl_Argumente}}"`
```

Comment:
The following fix could be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/nix-shell.md) page is updated:

```suggestion
`nix-shell --run "{{Befehl}} {{arg1 arg2 ...}}"`
```

Wrong syntax is used for zero or more arguments.